### PR TITLE
Fix Data Integration issues in update action

### PR DESCRIPTION
**Pull request by anupama-pathirage**

### CODE REVIEW

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/data/sql/Constants.java [hunk 1]
```diff
@@ -58,8 +58,6 @@
      * Constants for SQL DataTypes.
      */
     public static final class SQLDataTypes {
-        public static final String LONG = "LONG";
```
Comment (chanakaudaya) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/data/sql/Constants.java]: Can you elaborate on why these types are removed with this PR?

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/data/sql/client/UpdateWithGeneratedKeyColumns.java [hunk 2]
```diff
@@ -56,8 +57,8 @@
     public BValue execute(Context context) {
         BConnector bConnector = (BConnector) getArgument(context, 0);
         String query = getArgument(context, 1).stringValue();
-        BArray<BString> keyColumns = ((BArray<BString>) getArgument(context, 2));
-        BArray parameters = (BArray) getArgument(context, 3);
+        BArray parameters = (BArray) getArgument(context, 2);
```
Comment (chanakaudaya) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/data/sql/client/UpdateWithGeneratedKeyColumns.java]: Is this ordering change essential? Can you elaborate more?

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/data/sql/Constants.java (hunk 1 above)
Comment (anupama-pathirage) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/data/sql/Constants.java]: In the current implementation, only the standard jdbc sql types[1] can be used to define the type of the input/output parameters. Long, String etc are not generic sql types and those should be defined as BIGINT, VARCHAR. But still for any "Long" type supported DB, we can use BIGINT as parameter type when defining the parameter and get the same result.  In future we can add those DB specific types if needed. 
[1] http://docs.oracle.com/javase/6/docs/api/java/sql/Types.html

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/data/sql/client/UpdateWithGeneratedKeyColumns.java (hunk 2 above)
Comment (anupama-pathirage) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/data/sql/client/UpdateWithGeneratedKeyColumns.java]: For all the other SQL Connector actions the parameter order is SQLConnector, SQLQuery and Parameters. This is an overloaded method for the action "updateWithGeneratedKeys" which has one more additional string array parameter for 'keyColumns. To comply with the other actions initial 3 parameters were kept in the same order.

**Pull request merged**